src/Gui/DlgUnitsCalculator.ui

```diff
@@ -114,12 +114,17 @@ To add a calculation press Return in the value input field</string>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
-         <string>Scheme:</string>
+         <string>Unit System:</string>
```

Comment:
I am against this "title case". I think we should only capitalize the first letter.

`<string>Unit system:</string>`

Reply:
Thanks for the review. I changed it accordingly.